server.js

```diff
@@ -99,14 +103,56 @@ function handleWeather(request, response) {
 //   }
 }
 
-
-
-
 function Weather(data) {
   this.time = data.time;
   this.forecast = data.summary;
 }
 
+
+
+
+// SQL locations
+
+app.get('/location', (req,res) => {
+  
+  const SQL = 'SELECT * FROM location';
```

Comment:
You want to do two things here.

1)  Add a WHERE clause to your SQL variable.
`const SQL = 'SELECT * FROM location WHERE city = $1;`

2) Add a second variable for your SQL safe values.  `searched_city_variable` is the city that you have searched for.
`const sqlArr = [searched_city_variable];`